EIPS/eip-7830.md

```diff
@@ -29,8 +29,6 @@ Storage cost is already paid per contract byte.
 
 [EIP-170](./eip-170.md) specifies `MAX_CODE_SIZE` as 24576 bytes, and [EIP-3860](./eip-3860.md) specifies `MAX_INITCODE_SIZE` as `2 * MAX_CODE_SIZE` (49152 bytes).
 
-<!-- TODO: if profiling analysis shows we need to charge more for EOF analysis, this is where we can specify it.  Either globally or for 0xef00 contracs -->
```

Comment:
@shemnon @chfast @gumb0 do we expect more cost for EOF analysis? I doubt.

Reply:
Reminder: validation is only performed when a contract is brought in via a transaction. A factory create from an existing contract has no validation burden.

Reply:
Tracked in https://github.com/ipsilon/eof/issues/175